_posts/0000-01-02-bnaxway.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
Blah Blah Blah I still don't get this and still can't get excited about programming